tools/accounts-balance-checker/pkg/check/data.go

```diff
@@ -43,6 +44,7 @@ type AccountResponse struct {
 	Code  string `json:"code"`
 }
 
+// BalancesESDTResponse holds  the account esdt balances endpoint response
```

Comment:
remove one space between `holds` and `the`

Reply:
fixed